docs/sphinx/source/changelog/v2.1.0.rst

```diff
@@ -0,0 +1,54 @@
+**************************
+v2.1.0 (October XX, 2020)
+**************************
+
+API Changes
+-----------
+* Add :py:class:`~rdtools.analysis.RdAnalysis` class for single-line analysis. (:pull:`117`).
+
+
+Enhancements
+------------
+* Add new :py:mod:`~rdtools.analysis` module to focus on single-line analysis workflow
+  (:pull:`117`).
+* Add :py:class:`~rdtools.analysis.RdAnalysis` class for single-line analysis. 
+  Degradataion analysis is run by :py:meth:`~rdtools.analysis.RdAnalysis.sensor_analysis` 
```

Comment:
```suggestion
  Degradation analysis is run by :py:meth:`~rdtools.analysis.RdAnalysis.sensor_analysis` 
```

I make this typo all the time, I guess because our fingers are used to typing "data".